pkg/sdk/database_role_dto_builders.go

```diff
@@ -0,0 +1,110 @@
+package sdk
+
+func NewCreateDatabaseRoleRequest(
+	name DatabaseObjectIdentifier,
+) *CreateDatabaseRoleRequest {
+	s := CreateDatabaseRoleRequest{}
+	s.name = name
+	return &s
+}
+
+func (s *CreateDatabaseRoleRequest) WithOrReplace(orReplace bool) *CreateDatabaseRoleRequest {
+	s.orReplace = orReplace
+	return s
+}
+
+func (s *CreateDatabaseRoleRequest) WithIfNotExists(ifNotExists bool) *CreateDatabaseRoleRequest {
+	s.ifNotExists = ifNotExists
+	return s
+}
+
+func (s *CreateDatabaseRoleRequest) WithComment(comment *string) *CreateDatabaseRoleRequest {
+	s.comment = comment
+	return s
+}
+
+func NewAlterDatabaseRoleRequest(
+	name DatabaseObjectIdentifier,
+) *AlterDatabaseRoleRequest {
+	s := AlterDatabaseRoleRequest{}
+	s.name = name
+	return &s
+}
+
+func (s *AlterDatabaseRoleRequest) WithIfExists(ifExists bool) *AlterDatabaseRoleRequest {
+	s.ifExists = ifExists
+	return s
+}
+
+func (s *AlterDatabaseRoleRequest) WithRename(rename *DatabaseRoleRenameRequest) *AlterDatabaseRoleRequest {
+	s.clearForOneOf()
+	s.rename = rename
+	return s
+}
+
+func (s *AlterDatabaseRoleRequest) WithSet(set *DatabaseRoleSetRequest) *AlterDatabaseRoleRequest {
+	s.clearForOneOf()
+	s.set = set
+	return s
+}
+
+func (s *AlterDatabaseRoleRequest) WithUnsetComment() *AlterDatabaseRoleRequest {
```

Comment:
Because this one is a shortvut and set is handled with Set and then WithComment. But I have changed it in a following PR to SetComment also (because it's easier to use and more consistent, so I will change it also here).

Reply:
Done.